mix.exs

```diff
@@ -47,6 +47,7 @@ defmodule Rihanna.MixProject do
       {:ecto, ">= 2.0.0", optional: true},
       {:ecto_sql, ">= 3.0.0", optional: true},
       # Development tools
+      {:jason, ">= 1.1.2", only: :test},
```

Comment:
I suppose I could make `Jason` optional. If people use another JSON library they can set it on postgrex's configuration.

Reply:
Hmm. Will this fail if someone manages to install it on a project without Jason already? I think this should probably just be global and non-optional.

Reply:
Postgrex does not require a specific json library because it works with a few (via config option) and people use different ones. I'll make note of it when I add docs.

Reply:
Added note about json library to the readme in the section under "Without Ecto".